RELEASE_INFO-6.7.md

```diff
@@ -294,3 +294,4 @@ https://github.com/shopware/shopware/pull/12654 by ffrank913 fixes Incorrect fro
   * Removal of `controllerName` and `controllerAction` variables in templates
   * Deprecation of `Shopware\Core\System\SalesChannel\SalesChannel\SalesChannelContextSwitcher`
 * Upgrade notes: DB migration for the new category index, admin build target upgrade, profiler defaults
+* ProductStream IDs added to ElasticsearchProductDefinition
```

Comment:
this were the examples for 6.7.3.0

as we are preparing 6.7.5.0 already, this needs to be put at the top of this file